package/Dockerfile

```diff
@@ -1,4 +1,7 @@
 FROM scratch
 
+ARG K3S_DATA_DIR
+ENV K3S_DATA_DIR=${K3S_DATA_DIR}
+
```

Comment:
Same comment as the rke2 change - the image contents are extracted out to disk by rancher-system-agent and run on the host system. This Docker image is never run, so the args and env vars don't matter.